src/cryo_challenge/_map_to_map/map_to_map_distance_matrix.py

```diff
@@ -41,20 +99,19 @@ def run(config):
     label_key = config["data"]["submission"]["label_key"]
     user_submission_label = submission[label_key]
 
-    # n_trunc = 10
-    metadata_gt = pd.read_csv(config["data"]["ground_truth"]["metadata"])#[:n_trunc]
+    metadata_gt = pd.read_csv(config["data"]["ground_truth"]["metadata"])
 
     results_dict = {}
     results_dict["config"] = config
     results_dict["user_submitted_populations"] = (
         submission[submission_metadata_key] / submission[submission_metadata_key].sum()
     )
 
-    cost_funcs_d = {
-        "fsc": compute_cost_fsc_chunk,
-        "corr": compute_cost_corr,
-        "l2": compute_cost_l2,
-        "bioem": compute_bioem3d_cost,
+    map_to_map_distances = {
```

Comment:
@DSilva27 what do you think about this part here. Maybe there's a way to handle this mapping in the config etc and not have to have it hard coded here.

Reply:
I think we could have an equivalent dictionary as a global variable right after importing the functions. Something like

```
AVAILABLE_METRICS = {
    "metric_id": METRIC_CLASS
}
```

Reply:
I'll do that when I refactor things into utils and pipeline